packages/stockflux-watchlist/src/components/watchlist/Watchlist.js

```diff
@@ -36,17 +43,16 @@ const Watchlist = () => {
   const WINDOW_OFFSET = 5;
 
   const [options] = useOptions();
+  const notification = useNotification(NOTIFICATION_OPTIONS);
 
   const displayNotification = newSymbol => {
-    const alreadyInWatchlist = watchlist.includes(newSymbol);
-    showNotification({
-      message: {
-        symbol: newSymbol,
-        watchlistName: 'My Watchlist',
-        alreadyInWatchlist,
-        messageText: `${alreadyInWatchlist ? ' moved' : ' added'} to the top`
-      }
-    });
+    setNewSymbol(newSymbol);
+    if (notification) {
+      notification.launch({
+        url: 'notification.html',
+        timeout: 'never'
```

Comment:
The notification is currently appearing and never vanishing; should it be made to disappear after a programmed interval? Currently the user needs to drag it away to make it disappear which is not immediately obvious; is there a way to add an 'X' to show the user how they can close it?

Reply:
Thanks, Katriona, that's a very good point, removed it now altogether, which defaults to 3600 milliseconds. 
There is a way to add X button, but there's no such requirement in UX specs.